### PR TITLE
fix: typo in `<Dropdown>` component

### PR DESCRIPTION
**Pull request by hirehamir**
# Without this change

image

# With this change

image

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-import Link from "next/link";
+import React, { ReactNode, useState } from "react";
+import Select, { StylesConfig, createFilter, defaultTheme } from "react-select";
 import Button from "~/components/Button";
 import Input from "~/components/Input";
-import { api } from "~/utils/api";
 import TopMenu from "~/components/TopMenu";
-import Select, { createFilter, StylesConfig } from "react-select";
-import { defaultTheme } from "react-select";
+import { api } from "~/utils/api";
```
Comment (hirehamir) [inline src/pages/index.tsx:9]: We should look into import-sorting being part of formatting.

Comment (hirehamir) [reply src/pages/index.tsx:9]: I think it'll make diffs less cluttered.

Comment (elwerro) [reply src/pages/index.tsx:9]: I believe Wayne wanted a different sort for the import, that isn't alphabetical lol. Yesterday Wayne and I decided to go with a different dropdown component, so this is all going to change.

**Comment by elwerro** [comment]
We're going to use something similar to this https://tailwindui.com/components/application-ui/elements/dropdowns

File: src/pages/index.tsx (hunk 1 above)
Comment (meows) [reply src/pages/index.tsx:9]: The different sort should only affect Tailwind. It'd be nice to have auto cleanup of imports.

Comment (hirehamir) [reply src/pages/index.tsx:9]: Okay, I'll close this then, since it seems like we're abandoning this component.

**Pull request closed without merging**